### PR TITLE
Adding wheelchair detector in the tmux script.

### LINKED ISSUE #209: Add wheel chair detector to tracker
Needs to go here: https://github.com/strands-project/strands_perception_people/blob/indigo-devel/bayes_people_tracker/config/detectors.yaml

Needs to publish a pose array or use to_pose_array

### PR DESCRIPTION
**Pull request by Pandoro**
As discussed this morning in the meeting with @marc-hanheide . It's pretty much impossible for me to test this. So I'd leave it unmerged for now, but have it here for the moment we want give it a go when Werner is at HdB. (This then also fixes strands-project/strands_perception_people#209)

### CODE REVIEW

**Review by marc-hanheide — COMMENTED** [review]
to be tested by @Jailander

File: aaf_bringup/scripts/aaf_start.sh [hunk 1]
```diff
@@ -47,7 +47,7 @@ tmux select-window -t $SESSION:5
 tmux send-keys "DISPLAY=:0 roslaunch aaf_bringup aaf_navigation.launch map:=/opt/strands/map/aaf_deployment16-cropped.yaml topological_map:=aaf_deployment16"
 
 tmux select-window -t $SESSION:6
-tmux send-keys "DISPLAY=:0 roslaunch perception_people_launch people_tracker_robot.launch machine:=werner-left-cortex user:=strands"
+tmux send-keys "DISPLAY=:0 roslaunch perception_people_launch people_tracker_robot.launch machine:=werner-left-cortex user:=strands with_wheelchair_detector:=true wheelchair_machine:=werner-left-cortex wheelchair_user=strands"
```
Comment (Jailander) [inline aaf_bringup/scripts/aaf_start.sh]: `wheelchair_user=strands` should be replaced by `wheelchair_user:=strands` second time this happens to me

**Comment by marc-hanheide** [comment]
I fixed it.

**Comment by marc-hanheide** [comment]
@Jailander @Pandoro,  can you confirm that 12a60dc fixed it?

**Comment by Jailander** [comment]
Yes, please merge

**Pull request merged**